src/player.js

```diff
@@ -1426,7 +1435,9 @@ export default class Player extends FakeEventTarget {
    * @return {?ThumbnailInfo} - Thumbnail info
    */
   getThumbnail(time: number): ?ThumbnailInfo {
-    if (this._engine) {
+    if (this._externalThumbnailsHandler.isUsingVttThumbnails()) {
```

Comment:
it overnight dash thumbnails - confluence

Reply:
overnight?

Reply:
It also works during the day of course ... the comment was fixed